### ISSUE TITLE
[Issue]: Force Vertical Offset doesn't honor the Default Game Settings

### ISSUE THREAD

**Issue by Anakiev2** [emukidid/swiss-gc#910]
### Describe the bug

Setting "Force Vertical Offset" in "Default Game Settings" to some value is not loaded inside the "Current Game Settings". It's about r1712 but I've seen it in earlier revisions too.

### Steps to reproduce

Delete "/swiss/settings".
Set "Force Vertical Offset" in "Default Game Settings" to "0".
Save the settings.
Open a game and go to "Current Game Settings".
It's always "-3".

### Expected behavior

"Force Vertical Offset" inside "Current Game Settings" should be "0".

**Comment by Extrems** [comment]
Yeah, it's been that way ever since default game settings were introduced. The default is actually based on the AVE Compatibility setting.

**Comment by Extrems** [comment]
Fixed in 8fe49ec (r2091).
